design-docs/CLI-Orders.md

```diff
@@ -508,17 +449,79 @@ $ planet orders request \
 {"name":"test_order",...}
 ```
 
+## request-update
+
+It is a common use case to want to create an order that is just slightly
+different from another order. Maybe the order has the same toolchain but just
+new ids. Maybe the order has the same ids but just a different clip AOI. It
+would be nice to just update the old order request with the new information
+instead of having to start from scratch.
+
+Making this its own command frees the `request` command from having a ton of
+required options, which are a bit annoying from a user's perspective.
+
+
+### Interface
+
+Usage: planet orders request-update [OPTIONS] REQUEST
+
+REQUEST: order request to be updated. JSON can be str, filename, or stdin.
+Defaults to reading from stdin.
+
+Update an order request.
+
+This command creates a new order description from an existing order description,
+with certain entries changed according to the options specified. It outputs the
+updated order request, optionally pretty-printed.
+
+Support for updating an order description is provided however it
+has many limitations compared to what the Orders API supports. For creation
+of more advanced order requests, create the order description manually or
+use the Python library to aid in creating the order description.
+
+Note that if the order description contains a tool chain with more tools
+than just clip, using --clip will override the entire tool chain, not just
+the clip tool.
+
+Options:  
+
+  --clip JSON        -              Clip feature GeoJSON. Can be a json string,
+                                  filename, or '-' for stdin.  
+  --tools JSON         -           Toolchain JSON. Can be a json string,
+                                  filename, or '-' for stdin.  
+  --email BOOLEAN         -          Send email notification when order is
+                                  complete.  
+  --cloudconfig JSON     -         Credentials for cloud storage provider to
+                                  enable cloud delivery of data. Can be a json
+                                  string, filename, or '-' for stdin.  
+  --pretty                -        Format JSON output.
+
+### Usage Examples
+
+To create an order using an order id and override the item IDs:
+
+```
+$ planet orders get 49b8d32e-2fba-4924-bd38-f7344aa48d91 > \
+  planet orders request-update \
+  --id 20200922_183724_23_106a,20200922_183722_17_106a
+```
+  To create an order using an order id and clip to an AOI:
+```
+$ planet orders get 49b8d32e-2fba-4924-bd38-f7344aa48d91 > \
+  planet orders request-update --clip aoi.geojson
+```
+
 User Story: As a CLI user I would like to create a request for an order from a
 template, overriding the name.
 
 ```
-$ planet orders request --like - --name IAmACopy < request.json
```

Comment:
Why would you use the '<' here instead of just `planet orders request --like request.json --name IAmACopy`? The '<' is still beyond my command-line knowledge.

Reply:
yeah, really, there is no reason for using `< requests.json` instead of just using `requests.json`